test/mongoConnectionStrings.test.ts

```diff
@@ -5,53 +5,54 @@
 
 // The module 'assert' provides assertion methods from node
 import * as assert from 'assert';
-import { getDatabaseNameFromConnectionString } from '../src/mongo/mongoConnectionStrings';
+import { parseConnectionString } from '../src/mongo/mongoConnectionStrings';
 
-function testDatabaseFromConnectionString(connectionString: string, expectedDatabase: string | undefined) {
-    let database = getDatabaseNameFromConnectionString(connectionString);
-    assert.equal(database, expectedDatabase);
+function testParseConnectionString(connectionString: string, expectedDatabase: string[]) {
```

Comment:
The goal was to use existed tests for both functions, so I'm going to create a new tests for addDatabaseToConnectionString(aka getDatabaseConnectionStringByAccountConnectionString). I think that my tests changes may be useful for future development in case if any other part of the connection string will be needed, so what do you think if I make them and refactor testing function in a next way:
`
function testDatabaseItemFromConnectionString(connectionString: string, expectedDatabase: string[]) {
    let database = getDatabaseNameFromConnectionString(connectionString);
    assert.equal(database, expectedDatabase[2]);
}

suite(`mongoCollectionStrings`, () => {
    test(`getDatabaseItemFromConnectionString`, () => {
        testDatabaseItemFromConnectionString(...);
        ...
  `